README.md

```diff
@@ -1 +1,99 @@
-# Organizer-Ruby_Capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Catalog of my things
+
+> Tis project is a console app that will help users to keep a record of different types of things they own: books, music albums, movies, and games.
+
+## Description video
+
+https://user-images.githubusercontent.com/91022355/190726589-8b55ccdb-9266-4c09-8a81-720a65e58252.mp4
+
+## Built With
+
+- Major languages : `Ruby`
+- Frameworks : `Rspec`
+- Technologies used : `Git`, `Gitflows`, `Github`, `Rubocop` and `Rubygem`
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have a computer and internet connection
+- Have `Ruby` installed on your computer
+- Have a basic knowledge of `Ruby` and `OOP` concept
+- Have a general understanding of what testing is
+- Have `visual-studio code` or any other code editor installed on your computer.
+
+### Setup
+
+- In order to get a copy of this project you need to download it from https://github.com/jahongiry/Organizer-Ruby_Capstone-project
+- Extract the zipped file and open it in your code editor
+
+### Install
+
+- Run the command bellow in your terminal to get all required files
+
+```
+bundle install
+```
+
+### Run tests
+
+you can run one of the following command in your terminal
+
+1. Run testing
+
+```Ruby
+rspec spec
+```
+
+2. Run linters
+
+```Ruby
+> Rubocop --color
+> Rubocop -A
+```
+
+### Run the project
+
+`ruby main.rb`
+
+## Authors
+
+👤 **Husseini Mudi Profsain**
+
+- GitHub: [@githubhandle](https://github.com/Profsai)
+- Twitter: [@twitterhandle](https://twitter.com/profsain)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/profsain)
+
+👤 **Jahongir Yusupov**
+
+- GitHub: [@githubhandle](https://github.com/jahongiry)
+- Twitter: [@twitterhandle](https://twitter.com/@JahongirYusup13)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/@JahongirYusup13/)
+
+👤 **Sahar**
```

Comment:
- [ ] OPTIONAL
I think it will be a good idea if you add your full name (Sahar). This will make your ReadMe more professional.